### PR TITLE
MAYA122097 Make the word 'tip'bold

### PR DESCRIPTION
**Pull request by pierrebai-adsk**
Had to remove adjustable column from teh layout otherwise the text field would expand vertically a take a lot of vertical space.

### CODE REVIEW

File: lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ def fileOptionsTabPage(tabLayout):
     cmds.setParent(topForm)
     cmds.frameLayout(label=getMayaUsdLibString("kMayaRefDescription"))
     cmds.columnLayout(adjustableColumn=True)
-    cmds.text(align="left", label=getMayaUsdLibString("kMayaRefCacheToUSDDescription"))
+    cmds.text(align="left", wordWrap=True, label=getMayaUsdLibString("kMayaRefCacheToUSDDescription"))
```
Comment (seando-adsk) [inline lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py:253]: This column is still adjustable - does it work correctly here?

File: lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel [hunk 2]
```diff
@@ -115,8 +115,11 @@ proc fileOptionsTabPage(string $tabLayout)
 
     setParent $topForm;
     frameLayout -label `getMayaUsdLibString("kMayaRefDescription")`;
-    columnLayout -adjustableColumn true;
-    text -align "left" -label `getMayaUsdLibString("kMayaRefAddToUSDDescription")`;
+    // Note: if column is adjustable, then the label takes too much vertical space.
+    //       when set no adjuatable, the text does not take the full width. This
```
Comment (seando-adsk) [inline lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel:119]:
```suggestion
    //       when set not adjustable, the text does not take the full width. This
```

File: lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py (hunk 1 above)
Comment (pierrebai-adsk) [reply lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py:253]: I'll re-verify this panel.

Comment (pierrebai-adsk) [reply lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py:253]: In this dialog, the vertical space effect is less severe and appear natural. I would leave as it is now.

**Comment by pierrebai-adsk** [comment]
image

**Review by seando-adsk — APPROVED** [review]

**Comment by seando-adsk** [comment]
@pierrebai-adsk There are merge conflicts to fix.

**Review by seando-adsk — APPROVED** [review]

**Review by seando-adsk — APPROVED** [review]

**Pull request merged**